packages/file_selector/file_selector_web/pubspec.yaml

```diff
@@ -0,0 +1,32 @@
+name: file_selector_web
+description: Web platform implementation of file_selector
+homepage: https://github.com/flutter/plugins/tree/master/packages/file_selector/file_selector_web
+version: 0.7.0
+
+flutter:
+  plugin:
+    platforms:
+      web:
+        pluginClass: FileSelectorWeb
+        fileName: file_selector_web.dart
+
+dependencies:
+  file_selector_platform_interface: ^1.0.0
```

Comment:
Note, that there was a MV bump of `cross_file`, which I guess is getting reexported from this package. I think we might want to use it (it's a very minor change, but potentially backwards-incompatible)